main/src/library/TimeUnit.flix

```diff
@@ -26,3 +26,22 @@ enum TimeUnit with Eq {
     case Nanoseconds,
     case Seconds
 }
+
+mod TimeUnit {
+
+    ///
+    /// Returns the number of miliseconds for timeunit `u`
+    ///
+    pub def toMilliseconds(u: TimeUnit): Int32 =
+        let ms = match u {
```

Comment:
No need for the extra local variable

just do

def toMilliseconds(u: TimeUnit): Int32 = match u {
 ...